examples/php/greeter_server.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+require dirname(__FILE__) . '/../../src/php/lib/Grpc/MethodDescriptor.php';
+require dirname(__FILE__) . '/../../src/php/lib/Grpc/Status.php';
+require dirname(__FILE__) . '/../../src/php/lib/Grpc/ServerCallReader.php';
+require dirname(__FILE__) . '/../../src/php/lib/Grpc/ServerCallWriter.php';
+require dirname(__FILE__) . '/../../src/php/lib/Grpc/ServerContext.php';
+require dirname(__FILE__) . '/../../src/php/lib/Grpc/RpcServer.php';
+require dirname(__FILE__) . '/vendor/autoload.php';
+
+class Greeter extends Helloworld\GreeterServiceStub
+{
+    public function SayHello(
+        \Helloworld\HelloRequest $request,
+        \Grpc\ServerContext $serverContext
+    ): array {
+        $name = $request->getName();
+        $response = new \Helloworld\HelloReply();
+        $response->setMessage("Hello " . $name);
+        return [$response];
```

Comment:
Why does this `[$response]` need to be an array? Shouldn't it just be `return $response;`? Same for all the unary methods in the rest of the examples.

I see. So user is expecting to return `list($response, $initialMetadata, $status)` here? Then let's figure out what's a better way to do this.

As a user I expect to write `return $response;`. If I need to return initial metadata and status, perhaps we should do it via the `$serverContext`. Let's see how other languages do this.

Yea, check out how Ruby implemented the interop test [here](https://github.com/grpc/grpc/blob/master/src/ruby/pb/test/server.rb#L179-L185). The return value of the RPC method itself is just the response proto. Everything else, like the status code, message, metadata are all set to the `_call` object. For us here in PHP, we can do it via this `$serverContext` object.

Reply:
conform to the existing greeter client.
[list($response, $status) = $client->SayHello($request)->wait();](https://github.com/grpc/grpc/blob/b516dbfa21c5a2065f8a99a5890316d7da75d238/examples/php/greeter_client.php#L34)